src/app/modules/group/components/group-links/group-links.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+import { Router } from '@angular/router';
+import { GroupShortInfo } from '../../http-services/get-group-by-id.service';
+
+const MAX_ITEMS_DISPLAY = 3;
+
+@Component({
+  selector: 'alg-group-links',
+  templateUrl: './group-links.component.html',
+  styleUrls: [ './group-links.component.scss' ]
+})
+export class GroupLinksComponent {
+  @Input() items?: GroupShortInfo[];
+
+  maxItemsDisplay = MAX_ITEMS_DISPLAY;
+
+  get hasMoreItems(): boolean {
```

Comment:
This function is called more than 100 times for one page (I assume that's because it is a function and change detection is on). This is a constant anyway which maybe computed on onchange. The name is confusing anyway (same for moreItemsCount) , it sounds like a loop property like "hasNext()"

Reply:
Yes, it most well for onPush strategy, I'll move conditions to template

Reply:
Fixed